bindings/imap-codec-python/Cargo.toml

```diff
@@ -14,6 +14,19 @@ crate-type = ["cdylib"]
 doc = false
 
 [dependencies]
-imap-codec = { version = "2.0.0-alpha.2", path = "../../imap-codec", features = ["serde"] }
 pyo3 = "0.22.1"
 serde-pyobject = "0.4.0"
+
+[dependencies.imap-codec]
+version = "2.0.0-alpha.2"
+path = "../../imap-codec"
+features = [
```

Comment:
This is relevant for you, @HenningHolmDE.

This is the list of features we want (for now). `ext_condstore_qresync` is not (quite) ready. `ext_*_referrals` probably won't be ready soon (and is AFAIK a seldomly used extension that introduces quite some difficult questions.)

I think we should go with this. Maybe I manage to push the other features over the finish line soon.

Reply:
Thanks for letting me know. It probably does not change much on the level I'm currently working on.